libs/frontend/util-edit-text/src/lib/prosemirror-utils.ts

```diff
@@ -142,3 +171,12 @@ export const replaceMark = (markType: MarkType, attrs: Attrs | null = null): Com
     return true
   }
 }
+
+export const createLinkNode = (schema: Schema, href: string, target?: string) => {
+  const attrs = {
+    title: href,
+    href: href,
+    target: target ?? null,
+  }
+  return schema.text(href, [schema.mark('link', attrs), schema.mark('u')])
```

Comment:
Is there a difference between `schema.mark('link')` and `schema.marks.link.create()`?

Reply:
Yes, I think so! It's just that `schema.mark('link')` would throw an error immediately if "link" is not defined in the schema because it automatically calls `create()`, while `schema.marks.link` will be `undefined`, so no error will be thrown as long as we don't call `.create()`.